jsonschema-core.xml

```diff
@@ -1240,26 +1243,52 @@
                             same position, if any.
                         </t>
                         <t>
-                            Omitting this keyword has the same behavior as an empty schema.
+                            This keyword produces an annotation value which is the largest
+                            index to which this keyword applied a subschema.  The value
```

Comment:
Why is the result the index of the last applied item rather than the list of indices of applied items? This looks unsymmetrical with how it works for "properties". (If this something that got discussed before, I probably missed it, so any pointer or clarification would be welcome.)

Reply:
Because it's not possible to have gaps in the list.  There could be if `contains` were used, but as noted in that section, `contains` is not used.  And if it were, then its annotation would be a list of indices, but it still wouldn't be possible for `items` to ever produce a gap, so why bother?  Particularly if the list is very long.

Reply:
Ok, makes sense, thanks!